api/src/wfl/module/all.clj

```diff
@@ -81,6 +83,7 @@
                                     :release  release
                                     :uuid     (UUID/randomUUID)
                                     :version  version
+                                    :watchers (mapv prn-str watchers)
```

Comment:
I think we want just `pr-str` here.

Reply:
`watchers` is a `TEXT[]` in DB, so `pr-str` will always give us a TEXT[1] while `(mapv prn-str watchers)` gives TEXT[n], either works but I prefer latter as there are already stuff (one or a few emails per record) in prod DB, and latter doesn't require paying too much on migrations, what do you think?

Reply:
For example, latter will store something like:
```clojure
["hornet@broadinstitute.org","[\"EmailAddress\" \"hornet@broadinstitute.org\"]","[\"SlackChannel\" \"C026PTM4XPA\"]"]
```
into DB: 
```sql
{"hornet@broadinstitute.org","[\"EmailAddress\" \"hornet@broadinstitute.org\"]","[\"SlackChannel\" \"C026PTM4XPA\"]"}
```

Reply:
Yeah, FTR: I did not spot this wrinkle.  Sorry.